docs/release.md

```diff
@@ -39,33 +39,71 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, or about every 15 weeks beginning
+on the first Monday in January.  This is the same [release cycle](https://kubernetes.io/releases/release/) as Kubernetes, but KubeVirt will trail Kubernetes releases
+by 1 to 4 weeks.  Here's an example release cycle using the 2022 calendar:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Number | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-------------------------|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 0.55                    | 2 (January 10)        |                                             |
+| 15                  | 1.24               | 15 (April 12)    | 0.55                    | 16 (April 19)         |                                             |
+| 17                  | 1.25               | 1 (May 23)       | 0.56                    | 2 (May 30)            | KubeCon + CloudNativeCon EU likely to occur |
+| 32                  | 1.25               | 15 (August 22)   | 0.56                    | 16 (August 29) - 19 (September 26)        |                                             |
+| 34                  | 1.26               | 1 (September 5)  | 0.57                    | 2 (September 12)      | KubeCon + CloudNativeCon NA likely to occur |
+| 49                  | 1.26               | 14 (December 05) | 0.57                    | 15 (December 12) - 18 (January 9)    |                                             |
+
+Initial release candidates are cut **every 4 weeks.**
+With a 15 week cadence, there will be at least 3 release candidates:
+- v*-rc.0
+- v*-rc.1
+- v*-rc.2
+
+The stable branch will be created when the -rc.2 is tagged.
```

Comment:
Does this mean

| Week in cycle | Tag | Branch |
|:------------------:|:-----:|:----------:|
| 0 | - | - |
| 4 | rc.0 | - |
| 8 | rc.1 | - |
| 12 | rc.3 | release-N |
| 15 | vN | - |

Is this correct?

Reply:
@fabiand Great table(and worth having it in the document). This brings me to a question.
Does it mean we have only 12 weeks for dev work and 3 weeks of ?something? once we have the release branch? Or asked another way, what is the expected behavior of branch `release-N` and `main`?

Reply:
@fabiand yes.  I added the table.

Reply:
This means the feature development cycle for a release is 12 weeks.  Once we branch, the remaining 4 weeks are to backport bug fixes.

Reply:
@rthallisey  Sounds good to me. Can we mention it in the doc? Also good opportunity to dedicate these weeks to stabilizing our tests.